### PR TITLE
Extending explanation about port exclusions

### PR DESCRIPTION
**Pull request by DBAlberto**
Extending explanation about port exclusions, adding how to verify if this is configured, how to avoid one common error during exclusion configuration, and one scenario when another process is using the same port defined as probe port in the IP cluster resource in the AG role. 
It is added reference to “cluster core IP resource” since there is a reference using port 58888 on: 
sql-docs/azure-sql/includes/virtual-machines-ag-listener-configure.md

### CODE REVIEW

**Comment by PRMerger8** [comment]
@DBAlberto : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@adbadram - Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/virtual-machines/windows/hadr-cluster-best-practices.md [hunk 1]
```diff
@@ -61,8 +61,22 @@ For your SQL Server availability group or failover cluster instance, consider th
...
 `netsh int ipv4 add excludedportrange tcp startport=59999 numberofports=1 store=persistent`
+It is necessary for the port to be excluded not being used when the exclusion is configured, otherwise, the execution will fail with a message like “The process cannot access the file because it is being used by another process.”
+To confirm if the exclusions have been configured, we can check by using: `netsh int ipv4 show excludedportrange tcp`  
+By setting this exclusion for the AG role IP probe port, we could prevent an **Event ID: 1069** with status 10048, from Windows Failover cluster events, this is identified with event detail:
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/hadr-cluster-best-practices.md:68]:
```suggestion
Setting this exclusion for the AG role IP probe port should prevent events such as **Event ID: 1069** with status 10048. This event can be seen in the Windows Failover cluster events with the following message:
```

File: azure-sql/virtual-machines/windows/hadr-cluster-best-practices.md [hunk 2]
```diff
@@ -61,8 +61,22 @@ For your SQL Server availability group or failover cluster instance, consider th
...
 `netsh int ipv4 add excludedportrange tcp startport=59999 numberofports=1 store=persistent`
+It is necessary for the port to be excluded not being used when the exclusion is configured, otherwise, the execution will fail with a message like “The process cannot access the file because it is being used by another process.”
+To confirm if the exclusions have been configured, we can check by using: `netsh int ipv4 show excludedportrange tcp`  
+By setting this exclusion for the AG role IP probe port, we could prevent an **Event ID: 1069** with status 10048, from Windows Failover cluster events, this is identified with event detail: 
+```
+Cluster resource '<IP name in AG role>' of type 'IP Address' in cluster role '<AG Name>' failed.
+```
+An Event ID: 1069 with status 10048, can be identified from cluster logs with events like:
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/hadr-cluster-best-practices.md:72]:
```suggestion
An Event ID: 1069 with status 10048 can be identified from cluster logs with events like: 
```

**Pull request closed without merging**